en/docs/concepts/user-management/users.md

```diff
@@ -0,0 +1,58 @@
+# Users
+
+A user is the digital representation of a physical user who interacts with applications. A user can be a digital identity of a human or a machine. A user can be recognized by a number of attributes. These are called **user claims**. User claims are data that defines the user, i.e., anything the user is, owned by, or associated with.
```

Comment:
I think we have renamed "claims" to "attributes" in the new portals. If so, let's not use the term "claims" in the concept docs either